python/mqt/syrec/simulation_view/dialogs/simulation_run_json_export_dialog.py

```
@@ -0,0 +1,212 @@
+from ...logger_utils import log_info_to_console
+from ...message_box_utils import MessageBoxType, show_and_request_ok_in_optionally_cancellable_notification
+from ..workers.simulation_run_json_export_worker import ExportedBatchData, SimulationRunJsonExportWorker
+from .base_progress_dialog import BaseProgressDialog
+
+EXPORTED_SIM_RUNS_DATA_LABEL: Final[str] = (
+    "In total {n_exported_sim_runs:d} simulation runs were exported with {n_skipped_sim_runs:d} simulation runs being skipped"
+)
+
+
+class SimulationRunJsonExportDialog(BaseProgressDialog[SimulationRunJsonExportWorker]):
+    def __init__(self, parent: QtWidgets.QWidget):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)